lib/k8s/client/declarative.ex

```diff
@@ -0,0 +1,90 @@
+defmodule K8s.Client.Declarative do
+  @moduledoc """
+  Declarative functions for the K8s.Client module.
+  """
+
+  alias K8s.Client.Imperative
+  alias K8s.Client.Runner.Base
+
+  @last_applied_configuration_annotation "ex.kubernetes.io/last-applied-configuration"
```

Comment:
This probably shouldn't use anything on `kubernetes.io` even by appearance. I'd orient it around the GitHub origin of this library or just make it configurable, with a non-`kubernetes.io` default.

Maybe something like this, I'm sure there's a limit on max length to contend with. WDYT @coryodaniel? 

```suggestion
  @last_applied_configuration_annotation "k8s.corydaniels.github.com/last-applied-configuration"
```

Reply:
🤦 Sorry about mucking up your last name, Cory 😳